presto-main/src/main/java/com/facebook/presto/execution/QueryManagerConfig.java

```diff
@@ -337,6 +338,19 @@ public QueryManagerConfig setResourceGroupRunTimeInfoRefreshInterval(Duration re
         return this;
     }
 
+    public Duration getResourceGroupRunTimeInfoMinFreshness()
+    {
+        return resourceGroupRunTimeInfoMinFreshness;
+    }
+
+    @Config("resource-group-runtimeinfo-min-freshness")
+    @ConfigDescription("Resource Group Runtime Info Refresh Interval")
```

Comment:
No need to make each word capitalized.... It's a sentence. Same for `resourceGroupRunTimeInfoRefreshInterval`